src/controller/user/routes.ts

```diff
@@ -1,10 +1,12 @@
 import { FastifyInstance } from 'fastify'
 import { getUserById } from './getUserById'
 import { getUserByEmail } from './getUserByEmail'
+import { verifyJWT } from '../middlewares/verifyJwt'
 import { registerUser } from './registerUser'
 
 export async function userRoutes(app: FastifyInstance) {
   app.post('/user', registerUser)
   app.get('/user/:id', getUserById)
-  app.get('/user', getUserByEmail)
+  // app.get('/user', getUserByEmail)
+  app.get('/user', { onRequest: [verifyJWT] }, getUserByEmail)
```

Comment:
That's why your tests are failing. You changed the route to 'get user by email' to require a JWT token, but the tests are expecting a different behavior.

https://github.com/MatheusSanchez/orange-back/actions/runs/7679970747/job/20931460068#:~:text=%E2%9D%AF%20src/controller/user/getUserByEmail.spec.ts%20%20(2%20tests%20%7C%202%20failed)%20113ms


You can reproduce this locally running this:

`pnpm run pretest:e2e`
`pnpm run test:e2e`

**You have two options to fix that here:**

You can leave the 'get user by email' route as it is for now, and in the future (with another PR), we can add this new requirement and change the tests accordingly.
OR

You need to change the tests to require a JWT token. The tests can be found here: [getUserByEmail.spec.ts](https://github.com/MatheusSanchez/orange-back/blob/master/src/controller/user/getUserByEmail.spec.ts)
I can explain how these end-to-end tests are working if you need help.